02_configure_host.sh

```diff
@@ -87,7 +87,7 @@ if [ ${NUM_EXTRA_WORKERS} -ne 0 ]; then
         -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
         -e "provisioning_url_host=$PROVISIONING_URL_HOST" \
         -e "nodes_file=$EXTRA_NODES_FILE" \
-        -e "virtualbmc_base_port=$(( $VBMC_BASE_PORT + $NUM_MASTERS + $NUM_WORKERS + 1 ))" \
+        -e "virtualbmc_base_port=$(($VBMC_BASE_PORT + $NUM_MASTERS + $NUM_WORKERS))" \
```

Comment:
When using the "(( ))" operator, the "$" sign is unnecessary for the variables:
```suggestion
        -e "virtualbmc_base_port=$((VBMC_BASE_PORT + NUM_MASTERS + NUM_WORKERS))" \
```